src/docs/architecture_pantsd.md

```diff
@@ -38,7 +38,7 @@ Services
 
 A Pants Daemon process has several services that it polls in order. Every service runs in a separate thread, and can be paused, resumed or terminated. Services can communicate with each other.
 
-Examples of services are [`FSEventService`](https://github.com/pantsbuild/pants/blob/master/src/python/pants/pantsd/service/fs_event_service.py#14), which takes care of listening for watchman events, [`SchedulerService`](https://github.com/pantsbuild/pants/blob/master/src/python/pants/pantsd/service/scheduler_service.py#21), which takes care of listening responding to those events to keep a warm `Graph`, and [`PailgunService`](https://github.com/pantsbuild/pants/blob/master/src/python/pants/pantsd/service/pailgun_service.py#15), which listens to `SchedulerService` and manages the lifetime of a `PailgunServer` responsible for spawning pants runs when requested by clients, it takes a `DaemonPantsRunner` as one of its arguments, to use as a template to spawn pants runs when requested by the clients.
+Examples of services are [`SchedulerService`](https://github.com/pantsbuild/pants/blob/master/src/python/pants/pantsd/service/scheduler_service.py#21), which takes care of listening responding to those events to keep a warm `Graph`, and [`PailgunService`](https://github.com/pantsbuild/pants/blob/master/src/python/pants/pantsd/service/pailgun_service.py#15), which listens to `SchedulerService` and manages the lifetime of a `PailgunServer` responsible for spawning pants runs when requested by clients, it takes a `DaemonPantsRunner` as one of its arguments, to use as a template to spawn pants runs when requested by the clients.
```

Comment:
FYI the plan is to get rid of the old docsite on the main branch. We will still have the docsite with the 1.30 branch and will publish all updates from that branch.

Why remove on main? We wouldn't be able to keep the original docsite with all of these deletions. For example, I recently deleted the Go README, which was used for the docsite.

--

So, you'll either want to port to the new site, or cherry-pick this change into 1.30.